File: MPP/MPP_Iteratia3_C#/Common/responses/GetAllConcertsResponse.cs
using Common.model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common.responses
{
    [Serializable]
    public class GetAllConcertsResponse : Response
    {
        public List<ConcertDTO> Concerts;
        public GetAllConcertsResponse(List<ConcertDTO> C)
        {
            Concerts = C;
        }
    }
}

File: MPP/MPP_Iteratia3_Java/Networking/src/main/java/responses/GetAllConcertsResponse.java
package responses;

import Model.ConcertDTO;

import java.util.List;

public class GetAllConcertsResponse implements Response{
    private List<ConcertDTO> concerts;

    public GetAllConcertsResponse(List<ConcertDTO> con){
        concerts = con;
    }

    public List<ConcertDTO> getConcerts() {
        return concerts;
    }
}
